src/slate/classes.py

```diff
@@ -22,7 +22,10 @@
     from pdfminer.pdfparser import PDFPage
 except ImportError:
     from pdfminer.pdfpage import PDFPage
-import utils
+if PYTHON_3:
+    import slate.utils as utils
+else:
+    import utils
```

Comment:
Talking about extra spaces, I can fix some style issues in classes.py using pycodestyle. Is it OK for you?